ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py

```diff
@@ -62,6 +62,30 @@ def test_jax_numpy_eig(dtype_and_x, as_variable, native_array, num_positional_ar
     )
 
 
+# eigvals
+@handle_cmd_line_args
```

Comment:
Similar issue here there is a redefinition of the test down in the file. This also needs to be fixed for taking into account, `eigvalsh` instead of eigvals. This would also cause the lint checks to pass, and hopefully the Jax frontend tests too if you've written the test correctly.